src/umpire/util/MemoryMap.inl

```diff
@@ -59,7 +59,7 @@ MemoryMap<V>::~MemoryMap()
 template <typename V>
 template <typename... Args>
 std::pair<typename MemoryMap<V>::Iterator, bool>
-MemoryMap<V>::get(void* ptr, Args&&... args) noexcept
+MemoryMap<V>::get(Key ptr, Args&&... args) noexcept
```

Comment:
Was the `void*` just laziness before? Or does this introduce any real changes?

Reply:
It's the same thing. `using Key = void*;`